usaspending_api/common/elasticsearch/elasticsearch_sql_helpers.py

```diff
@@ -12,17 +12,12 @@ def verify_database_view_exists(view_name: str) -> bool:
     return view_name in list_of_available_views
 
 
-def ensure_transaction_etl_view_exists(force: bool = False) -> None:
-    """
-    The view is used to populate the Elasticsearch transaction index.
-    This function will ensure the view exists in the database.
-    """
+def ensure_view_exists(view_name, force: bool = False):
```

Comment:
If you do another round of changes would be nice to have type hint for `view_name` as well since there is one for `force`.